HumanResourceManagementSystem/src/main/java/com/HRmanagementsystem/dao/AdminDaoImpl.java

```diff
@@ -164,6 +164,30 @@ else if (res > 0 && accountActivation == 1)
         return messageString;
     }
 
+    /**
+     * Checks if a given username exists in the 'admin' database table.
+     *
+     * @param username The username to check for existence.
+     * @return {@code true} if the username exists in the database, {@code false} otherwise.
+     * @throws RuntimeException (implicitly) if a database error occurs (logged to console).
+     */
+    public boolean isUserExist(String username) {
```

Comment:
doesUserExist() instead of isUserExist()

Reply:
Done